README.md

```diff
@@ -7,13 +7,13 @@ Python API to the National Immunization Survey (NIS) data.
 ## Getting started
 
 - This a poetry-enabled project.
-- Use `nisapi.cache_all_datasets()` to download, clean, and cache data. Then `nisapi.get_nis()` to get a lazy dataframe pointing to that locally cached data.
-- See `scripts/demo.py` for an example of how to cache and query the data.
-- See `scripts/clean_example.py` for an example of a script that you could run while iteratively developing the cleaning code in `nisapi/clean.py`.
-  - Copy `scripts/secrets_template.yaml` to `scripts/secrets.yaml` and fill out the `app_token`.
-  - You will need to acquire a [Socrata app token](https://support.socrata.com/hc/en-us/articles/210138558-Generating-App-Tokens-and-API-Keys).
-- See `scripts/demo_cloud.py` for a demo of how the data could be downloaded, cleaned, uploaded to Azure Blob Storage, and then downloaded from there.
-  - Copy `scripts/secrets_template.yaml` to `scripts/secrets.yaml` and fill out the `azure:` keys.
+- Get a [Socrata app token](https://support.socrata.com/hc/en-us/articles/210138558-Generating-App-Tokens-and-API-Keys). Copy `scripts/secrets_template.yaml` to `scripts/secrets.yaml` and fill out the `app_token`.
```

Comment:
Even after reading "What is an app token?" in the link provided, I still am not sure I understand what a Socrata app token is. Sorry I can be dense about these things. This is something for us to discuss, not a critique of the PR. But I'm mentioning it here because it does weaken my understanding of the code that follows to some degree. Same goes for a lot of the commands that look up and modify paths on a machine's file system.

Reply:
The app token is a secret string that tells data.cdc.gov who is requesting the data. If you don't pass an app token with your request, your data download will be slower.